packages/kbn-check-mappings-update-cli/current_mappings.json

```diff
@@ -2097,6 +2097,32 @@
       }
     }
   },
+  "composite-slo": {
+    "dynamic": false,
+    "properties": {
+      "id": {
+        "type": "keyword"
+      },
+      "name": {
+        "type": "text"
+      },
+      "budgetingMethod": {
+        "type": "keyword"
+      },
+      "compositeMethod": {
+        "type": "keyword"
+      },
+      "sources": {
+        "properties": {
+          "id": { "type": "keyword" },
+          "revision": { "type": "integer" }
+        }
+      },
+      "tags": {
+        "type": "keyword"
+      }
+    }
+  },
```

Comment:
Obligatory question: are all of these mapped fields going to be power search functionality? (nothing can be removed for the foreseeable future)

Reply:
Yes i will have to search with these fields :)